apps/services/auth/ids-api/src/app/delegations/test/delegations-filters.spec.ts

```diff
@@ -165,24 +165,23 @@ describe('DelegationsController', () => {
     })
 
     let res: request.Response
-    it(`GET ${path} returns verified response`, async () => {
-      res = await server
-        .get(path + `?delegationType=${AuthDelegationType.LegalRepresentative}`)
-        .set('X-Query-From-National-Id', testCase.fromLegalRepresentative[0])
+    it(`POST ${path} returns verified response`, async () => {
+      res = await server.post(path).send({
+        fromNationalId: testCase.fromLegalRepresentative[0],
+        delegationTypes: [AuthDelegationType.LegalRepresentative],
+      })
 
-      expect(res.status).toEqual(200)
+      expect(res.status).toEqual(201)
```

Comment:
A RPC should respond with 200 OK, NestJS by default responds 201 for POST but you can either use our `@island.is/nest/swagger` `Documentation` decorator to set `response.status` as `200` or NestJS `HttpCode` decorator.